components/bl-multiple-select-component/README.md

```diff
@@ -0,0 +1,33 @@
+# Multiple Select
+
+Multiple Select is a component of Backendless UI-Builder designer. This component allows you to handle multiple selections.
+
+## Usage
+
+Add a component to your page and add the options to the component.
+
+### Component Elements
+
+<dl>
+<dt>Multiple select field</dt>
+<dd>This is the field that displays all the options you selected.</dd>
+<dt>Options</dt>
+<dd>A drop down list that displays all the options that you specify in the `Options` propertie.</dd>
+</dl>
+
+### Component Properties
+
+  Name         | Type       | Default value       | Description
+ --------------|------------|---------------------|-------------------------------------------------------------------------------------
+  Disable      | bool       | false               | This property allows you to disable component.
+  Placeholder  | string     | 'Placeholder'       | This property allows you to select a label of component.
+  Options      | { label: 'string', value: 'string' } |           | This property allows you to add items to be displayed in the drop down list. It's displayed on the component's logic page.
```

Comment:
What if some certain string values we describe as `'any string'`, but it just a typeof string we write `<string>`?